frontend/packages/dev-console/src/components/add/AddCardSection.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Switch, Tooltip } from '@patternfly/react-core';
+import { PageLayout } from '@console/shared';
+import { useUserSettings } from '@console/shared/src/hooks/useUserSettings';
+import { AddActionGroup, ResolvedExtension, AddAction } from '@console/dynamic-plugin-sdk';
+import {
+  filterNamespaceScopedUrl,
+  getActionGroups,
+  getSortedExtensionItems,
+} from '../../utils/add-page-utils';
+import { useAccessFilterExtensions } from '../../hooks/hooks';
+import { AddGroup } from '../types';
+import AddCard from './AddCard';
+import useContainerWidth from './useContainerWidth';
+import { MasonryLayout } from './MasonryLayout';
+import './AddPage.scss';
+
+type AddCardsSectionProps = {
+  title: string;
+  namespace: string;
+  hintBlock?: React.ReactNode;
+  addActionExtensions: ResolvedExtension<AddAction>[];
+  addActionGroupExtensions: ResolvedExtension<AddActionGroup>[];
+};
+
+const AddCardsSection: React.FC<AddCardsSectionProps> = ({
+  title,
+  namespace,
+  hintBlock,
+  addActionExtensions,
+  addActionGroupExtensions,
+}) => {
+  const { t } = useTranslation();
+  const [showDetails, setShowDetails] = useUserSettings(
+    'console.addPage.showDetails',
+    'show',
+    true,
+  );
+  const filteredAddActionExtensions: ResolvedExtension<AddAction>[] = filterNamespaceScopedUrl(
+    namespace,
+    useAccessFilterExtensions(namespace, addActionExtensions),
+  );
+
+  const sortedActionGroup: ResolvedExtension<AddActionGroup>[] = getSortedExtensionItems(
+    addActionGroupExtensions,
+  );
+
+  const addGroups: AddGroup[] = getActionGroups(
+    filteredAddActionExtensions.filter(
+      (filterActionGroup) => filterActionGroup.properties.id !== 'import-from-samples',
```

Comment:
Instead of filtering this action here we should check if we can just remove this action from the `console-extensions.json`, or?

Reply:
I was thinking on the same lines. Maybe we can have a separate extension for the samples.